firstvoices/backend/search/signals/dictionary_entry_signals.py

```diff
@@ -18,87 +19,107 @@
     update_translation,
 )
 from backend.search.utils.constants import ES_RETRY_POLICY
-from firstvoices.celery import link_error_handler
+from firstvoices.celery import check_celery_status, link_error_handler
 
 
 # Signal to update the entry in index
 @receiver(post_save, sender=DictionaryEntry)
 def request_update_dictionary_entry_index(sender, instance, **kwargs):
+    check_celery_status("update_dictionary_entry_index", instance.id)
     if DictionaryEntry.objects.filter(id=instance.id).exists():
-        update_dictionary_entry_index.apply_async(
-            (instance.id,),
-            link_error=link_error_handler.s(),
-            retry=True,
-            retry_policy=ES_RETRY_POLICY,
+        transaction.on_commit(
+            lambda: update_dictionary_entry_index.apply_async(
+                (instance.id,),
```

Comment:
In theory these retries/the retry policy could be removed as they were addressing the race condition, but I had left them in in case the tasks fail for another reason. If the team would rather they be removed I can remove them.

Reply:
+1 for keeping. Can't hurt, might help